src/uk/gov/hmcts/contino/AppPipelineDsl.groovy

```diff
@@ -40,6 +40,12 @@ class AppPipelineDsl extends CommonPipelineDsl implements Serializable {
     config.crossBrowserTest = true
   }
 
+  void enableParallelCrossBrowserTest(List<String> browsers = ['chrome', 'firefox', 'safari', 'microsoft'], int timeout = 120) {
+    config.crossBrowserTestTimeout = timeout
+    config.parallelCrossBrowserTest = true
```

Comment:
this property looks redundant to me. If parallelCrossBrowsers are set, doesn't it mean we are asking for parallel execution.  Also, can we rename this method to  something like. 

`enableCrossBrowserTest(List<String> parallelBrowsersList = ['chrome', 'firefox', 'safari', 'microsoft'], int timeout = 120)`

Reply:
Good call about the redundant `config.parallelCrossBrowserTest` - I'll remove it.

Reply:
But I might not have explained myself very well in your previous question about overloading this method. Unless I'm misunderstanding your point, I believe your method rename would not allow services to use a default browser list. This would require each service to configure that themselves even though there's likely be a lot of repetition, which is all I'm trying to avoid. It would also need a bit more understanding from each team as to how to configure it correctly, rather than being a bit more plug-and-play like I'm hoping.

So this scenario with your suggested rename wouldn't be valid, as it won't know which method to use:
```
void enableCrossBrowserTest(int timeout = 120) { 
... 
}

void enableCrossBrowserTest(List<String> parallelBrowsersList = ['chrome', 'firefox', 'safari', 'microsoft'], int timeout = 120) { 
...
}
```
```
enableCrossBrowserTest()
```

Reply:
No, i meant we have only one method which is may be so that it doesn't break existing scripts and we provide this behaviour to everyone. Unless u see a challenge in making parallel execution a problem to all repos.

`void enableCrossBrowserTest(int timeout = 120, List<String> parallelBrowsersList = ['chrome', 'firefox', 'safari', 'microsoft']) {
`

Reply:
(just for the benefit of this thread) 
as we agreed @adusumillipraveen I'll leave the PR as it is, start getting some benefit from the changes, and in the meantime I’ll still look to get other QA's thoughts and create a further PR to either deprecate the old way or overload the existing method as you've suggested here